dockerhub-readme/templates/base.md.j2

```diff
@@ -93,8 +95,12 @@ Many users do not need a specific platform combination but would like to ensure
 ## Prerequisites
 
 - NVIDIA Pascal™ GPU architecture or better
-- CUDA [10.1/10.2/11.0](https://developer.nvidia.com/cuda-downloads) with a compatible NVIDIA driver
-- Ubuntu 16.04/18.04 or CentOS 7
+- CUDA [10.1/10.2/11.0{{ "/11.2" if is_stable }}](https://developer.nvidia.com/cuda-downloads) with a compatible NVIDIA driver
+{% if is_ngc %}
+- Ubuntu 18.04 or CentOS 7
```

Comment:
Waiting to hear from NGC about additional operating systems to add.

Reply:
They approved adding Ubuntu 20.04 & CentOS 8

Reply:
updated in 04eef7d